pipeline_steps/common.groovy

```diff
@@ -22,10 +22,11 @@ def install_ansible(){
     source .venv/bin/activate
 
     # These pip commands cannot be combined into one.
-    pip install -U six packaging appdirs
-    pip install -U setuptools
     pip install 'pip==9.0.1'
```

Comment:
You may have to set the index-url here to ensure this comes from pypi though, so:

pip install --index-url https://pypi.python.org/simple 'pip==9.0.1 setuptools==33.1.1'